src/player.cpp

```diff
@@ -656,11 +678,11 @@ void Player::addStorageValue(const uint32_t key, const int32_t value, const bool
 	}
 }
 
-bool Player::getStorageValue(const uint32_t key, int32_t& value) const
+bool Player::getStorageValue(const uint32_t key, std::string& value) const
 {
 	auto it = storageMap.find(key);
 	if (it == storageMap.end()) {
-		value = -1;
+		value = "-1";
```

Comment:
Same as two above.

Reply:
Using "-1" as the default value gives me the advantage that in the case of later projection to int I get an easy to compare value in lua/c++. What could I use instead?

Reply:
Empty string